_shows/07_08/cross_stitching.md

```diff
@@ -17,6 +17,17 @@ crew:
     name: unknown
 
 published: true
+
+links:
+  - type: Review
+    href: http://broadwaybaby.com/shows/cross-stitching/25085
+    snapshot: JWfRM
+    publisher: Broadway Baby 
+    author: Natasha Long
+    title: "Cross-Stitching"
+    date: 2008-08-18
+    stars: 3
+    quote: "In the end, Cross-Stitching is much more about two people attempting to sort out their feelings for one another than an individual's struggle with gender identity. "
```

Comment:
Weird characters in here. Either do a multi-line with `>` or escape quotes `\"`.